game/templates/game/board.html

```
@@ -479,7 +479,7 @@
             style="position:relative; width:min(95vw, 1400px); height:90vh; background:#13161e; border-radius:12px; overflow:hidden; padding-top:48px;">
             <button type="button" aria-label="Close rulebook" onclick="document.getElementById('rulebookModal').style.display='none'"
                 style="position:absolute; top: 16px; right:16px; z-index:10; background:#333; border:none; color:#fff; width:32px; height:32px; border-radius:50%; cursor:pointer; font-size:1rem;">✕</button>
-            <iframe src="/rules/" style="width:100%; height:100%; border:none;"></iframe>
+            <iframe id="rulebookFrame" src="/rules/" style="width:100%; height:100%; border:none;"></iframe>
```

HTMLHint (1.9.2)

[warning] A <iframe> element must have an accessible name.

(frame-title-require)